### DISCUSSION TITLE
Maximise window on Double click Toolbar

### DISCUSSION THREAD

**Discussion by oerix** [zen-browser/desktop#2692] [Ideas]
Just wondering, is it possible to implement a feature where you double click the Toolbar/Address bar, then the window maximises fully, and then restores back to original sizing when you double click again?

Basically, how it works on other Chromium based browsers, not sure if this is a Firefox limitation.

Thanks

**Comment by thebromie** [comment]
I second this! 
Expected Behavior:
https://github.com/user-attachments/assets/a6116552-99fe-4ef7-aea6-dcdf1e6f5575

Current Behavior:
https://github.com/user-attachments/assets/aea03d11-70b8-478c-b035-99f93d5a128e

**Comment by PetiPandaRou** [comment]
Very painful, all applications on macOS are working as the expected behavior, except Zen Browser.
+1 here, hope it will be resolve soon 🙏 

Thanks in advance
